velox/docs/functions/spark/string.rst

```diff
@@ -164,6 +164,16 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
         SELECT rtrim('kr', 'spark'); -- "spa"
 
+.. spark:function:: space(count) -> varchar
+
+    Returns a string consisting of `n` spaces. ::
```

Comment:
tip: `n` should be changed to ``count``, the same as parameter name.

Reply:
done